Formula/wtfutil.rb

```diff
@@ -0,0 +1,25 @@
+class Wtfutil < Formula
+  desc "Terminal based search engine for bash commands"
+  homepage "The personal information dashboard for your terminal."
```

Comment:
Hi, thanks for the formula! 

I think that:
- `homepage` should be `"https://wtfutil.com"`
- `desc` should be `"The personal information dashboard for your terminal"` (with no period at the end)

And you should be good.

Reply:
Sure, I will do it